### ISSUE TITLE
.b3dm tile rendering support

### ISSUE THREAD

**Issue by datwelk** [maplibre/maplibre-native#2725]
Add support for rendering .b3dm tiles according to the 3D tiles standard (https://github.com/CesiumGS/3d-tiles) 

Would this be possible using a custom layer and geometry source?
On a high level, what would be the required steps?
I'd be interested to dive into this a bit more, but would need some pointers of where to start.

**Comment by louwers** [comment]
Some previous discussion: https://github.com/maplibre/maplibre/discussions/329

**Comment by datwelk** [comment]
Thanks @louwers. Is that discussion an accurate representation of the current status, or have there been further discussions / developments aside from there?

**Comment by louwers** [comment]
There have been no developments in this area since.

**Comment by louwers** [comment]
@datwelk 3D Model support is scheduled as a subject for the next TSC Meeting.

11 September, 7 PM to 8 PM CEST https://zoom.us/j/97047175665?pwd=8C1EUbqXoNdUBJdmkfuhrb45PuxjzJ.1

**Comment by datwelk** [comment]
Thanks @louwers! Is is possible to attend that meeting as a non TSC member? In a more passive role, just to listen in on the discussions

**Comment by louwers** [comment]
Absolutely, they are open to anyone, we don't have formal TSC membership.

**Comment by mtpdog** [comment]
> 3D Model support is scheduled as a subject for the next TSC Meeting.

Please don't forget to discuss a 'height above ground' option for 3D Models. This can be useful in some cases.

**Comment by datwelk** [comment]
> Absolutely, they are open to anyone, we don't have formal TSC membership.

Are the notes available somewhere @louwers?
